pom.xml

```diff
@@ -81,6 +81,11 @@
                     <compilerArgs>
                         <arg>-saveSource</arg>
                     </compilerArgs>
+                    <!--
+                    Draft and pretty mode are turned on so that beginners can try out debugging
```

Comment:
Would be better to rely on source maps and keep the code size reasonably small as it affects SEO. But this PR only adds the comment, so 👍

Reply:
This is actually an old comment that you removed in your PR, i'm just restoring it as the explanation for why we have (or had) this set
https://github.com/gwtproject/gwt-site-webapp/commit/27070768ae49c048535358a03c70a5b9d84ddf98#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8L130-L134

I suspect that optimized js ends up being different enough that sometimes sourcemaps can't really "fix" where each line of code goes - like if a function is inlined, "stepping in" doesn't make sense any more. I'm not sure I agree with this perspective, but there are some easier ways to bring the size down for the moment if we're concerned about that.